javascript/modules/client/README.md

```diff
@@ -441,25 +677,326 @@ console.log(proposals);
 
 ## Multisig governance plugin client
 
+```ts
+import {
+  ClientMultisig,
+  Client,
+  Context
+  ContextParams,
+  ContextPlugin
+} from "@aragon/sdk-client";
+
+const contextParams: ContextParams = {
+  // network id or name
+  network: 31337,
```

Comment:
Shouldn't this be renamed to `networkId` ?

Reply:
I'm not sure if `networkId` makes sense, right now you can either pass the id or the network name, for example you can use either `rinkeby` or `4` with the  same result

Reply:
Does ContextParams support `Networkish` from ethers? If so, that's fine